packages/virtualized-lists/Lists/VirtualizedList.js

```diff
@@ -1969,7 +1970,9 @@ class VirtualizedList extends StateSafePureComponent<Props, State> {
 
 const styles = StyleSheet.create({
   verticallyInverted: {
-    transform: [{scaleY: -1}],
+    transform:
+      // It's important to invert the Y AND X axis to prevent a react native issue that can lead to ANRs on android 13
+      Platform.OS === 'android' ? [{scaleX: -1}, {scaleY: -1}] : [{scaleY: -1}],
```

Comment:
Perhaps just use `[{scale: -1}]` or `[{rotate: '180deg'}]` to simplify?

Reply:
Adressed!